taichi/program/kernel.cpp

```diff
@@ -196,10 +196,6 @@ int64 Kernel::get_ret_int(int i) {
   }
 }
 
-void Kernel::mark_arg_return_value(int i, bool is_return) {
-  args[i].is_return_value = is_return;
```

Comment:
Thanks! Could you also remove is_return_value here?

Reply:
Good catch! Done